CorsixTH/Lua/world.lua

```diff
@@ -2708,3 +2708,32 @@ function World:resetSideObjects()
     end
   end
 end
+
+--[[ When placing doors and objects the passable tiles need to be checked for overlapping
+passable tiles. This presents problems with objects like Bench where the passable tile
+is not for exclusive use of the Bench (another object can share that same tile)
+the footprint.shareable differentiates shareable passable tiles, and exclusive use
+passable tiles (the norm for most objects)]]
+--! param x - x map tile position
```

Comment:
Similarly there should be no space between `!` and `param` or `!` and `returns`. `returns` should be `return` and the type should be in round brackets.